src/components/ToggleSection.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { motion } from 'framer-motion'
+import { ReactNode, useState } from 'react'
+import styled from 'styled-components'
+
+import Toggle from './Inputs/Toggle'
+
+interface ToggleSectionProps {
+  title: string
+  children: ReactNode | ReactNode[]
```

Comment:
`ReactNode` encompasses the use of single of multiple children (no need for `ReactNode[]` afaik)

Reply:
That's what I thought too when I originally started using this pattern (and that it sucked to have to do this lol) - but it seems I didn't look further, and that `ReactFragment` is the type in the `ReactNode` type alias that enables multiple children :D 

`type ReactFragment = {} | (boolean | any[] | ReactChild)[]`

I'll remove the list variants :+1: